ssr/render.ts

```diff
@@ -56,15 +67,23 @@ const lazy = (creator) => {
 const clientBuildRoot = path.resolve(dirname, "../../client/build");
 
 const readBuildHTML = lazy(() => {
-  const html = fs.readFileSync(
-    path.join(clientBuildRoot, "index.html"),
-    "utf-8"
-  );
+  let html = fs.readFileSync(path.join(clientBuildRoot, "index.html"), "utf-8");
   if (!html.includes('<div id="root"></div>')) {
     throw new Error(
       'The render depends on being able to inject into <div id="root"></div>'
     );
   }
+  const scripts: String[] = [];
```

Comment:
```suggestion
  const scripts: string[] = [];
```

Uppercase `String` is the wrapper object, which is bad smell.